src/components/OptionsToolbar/OptionsToolbar.js

```diff
@@ -0,0 +1,639 @@
+          )}
+          {shouldShowButton.showSaveAsPNGButton && (
+            <li
+              onClick={() => {
+                this.setState({ activeMenu: undefined })
+                this.saveChartAsPNG()
+              }}
+            >
+              <Icon type="download" /> Download as PNG
+            </li>
+          )}
+          {shouldShowButton.showCopyButton && (
+            <li
+              onClick={() => {
+                this.setState({ activeMenu: undefined })
+                this.copyTableToClipboard()
+              }}
+              // className={`chata-toolbar-btn${
+              //   this.state.copiedTable === true ? ' green' : ''
+              // }${this.state.copiedTable === false ? ' red' : ''}`}
+              // data-tip="Copy table to clipboard"
+              // data-for={`chata-toolbar-btn-copy-tooltip-${this.props.id}`}
+            >
+              <Icon type="copy" /> Copy table to clipboard
+            </li>
+          )}
+          {shouldShowButton.showSQLButton && (
+            <li
+              onClick={() => {
+                this.setState({ activeMenu: undefined })
+                this.copySQL()
+              }}
+              // className={`chata-toolbar-btn${
+              //   this.state.copiedSQL === true ? ' green' : ''
+              // }${this.state.copiedSQL === false ? ' red' : ''}`}
+              // data-tip="Copy generated query to clipboard"
+              // data-for={`chata-toolbar-btn-copy-sql-tooltip-${this.props.id}`}
+            >
+              <Icon type="copy" /> Copy generated query to clipboard
+            </li>
+          )}
+          {shouldShowButton.showCreateNotificationButton && (
+            <li
+              onClick={() => {
+                this.setState({ activeMenu: undefined })
+                this.props.onNewNotificationCallback(this.props.originalQuery)
+              }}
+            >
+              <Icon
+                style={{ verticalAlign: 'middle', marginRight: '3px' }}
+                type="notification"
+              />
+              Create a notification from this query
+            </li>
+          )}
+        </ul>
+      </div>
+    )
+  }
+
+  renderToolbar = () => {
+    const displayType = _get(this.props.responseRef, 'state.displayType')
+    const response = _get(this.props.responseRef, 'props.queryResponse')
+
+    const shouldShowButton = {
+      showFilterButton:
+        isTableResponse(response, displayType) &&
+        _get(response, 'data.data.rows.length') > 1,
+      showCopyButton:
+        isTableResponse(response, displayType) &&
+        !!_get(response, 'data.data.rows.length'),
+      showSaveAsCSVButton:
+        isTableResponse(response, displayType) &&
+        !!_get(response, 'data.data.rows.length'),
+      showSaveAsPNGButton: CHART_TYPES.includes(displayType),
+      showHideColumnsButton:
+        this.props.autoQLConfig.enableColumnVisibilityManager &&
+        // !isAggregation(response) &&
```

Comment:
remove?

Reply:
this is for the feature I mentioned in slack. I'll be enabling it soon so want to keep

Reply:
ohh okay. sounds good